gitfile/scmprovider_github.go

```diff
@@ -36,8 +36,10 @@ func (d *GitHubScmProvider) Detect(repoUrl, filepath, ref string) (bool, string,
 	param := req.Param{}
 	if ref != "" {
 		param["ref"] = ref
+		v = append(v, param)
 	}
-	r, e := req.Get(fmt.Sprintf(GithubAPITemplate, m["repoUser"], m["repoName"], filepath), param)
+
+	r, e := req.Get(fmt.Sprintf(GithubAPITemplate, m["repoUser"], m["repoName"], filepath), v...)
```

Comment:
errors are usually named `err`. I would stick to conventions

Reply:
ok